### DISCUSSION TITLE
Backspace should delete the current input character and focus to the previous one

### DISCUSSION THREAD

**Discussion by reubennn** [ritikbanger/react18-input-otp#22] [Ideas]
You should add an additional prop so that you can toggle backspace behaviour. Existing behaviour is that backspace will remove the current character but remain focused on the current input. You have to press backspace again to go to the previous character. So I have to backspace twice to get to the previous character instead of once.

I would like behaviour that when you press backspace, it remove the current character input and focuses on the previous one.

**Comment by reubennn** [comment]
https://www.npmjs.com/package/react-verification-input is a good example of the behaviour I'm discussing here.

**Comment by ritikbanger** [comment]
I will have a look.

**Comment by ritikbanger** [comment]
This is fixed in the latest version
